npm/linux.Dockerfile

```diff
@@ -2,12 +2,36 @@ FROM mcr.microsoft.com/oss/go/microsoft/golang:1.23 AS builder
 ARG VERSION
 ARG NPM_AI_PATH
 ARG NPM_AI_ID
+RUN apt-get update && apt-get install -y iptables ipset ca-certificates conntrack grep && apt-get autoremove -y && apt-get clean
```

Comment:
make this a separate build stage to keep the concerns distinct and so it can parallelize - let the Go builder only build Go and we'll make an iptables target to get those bits.

move to something like 
```Dockerfile
FROM mariner-core AS iptables
RUN tdnf install -y iptables
```

then change your copies to `--from iptables`